homeassistant/components/growatt_server/sensor.py

```diff
@@ -252,6 +252,160 @@ class GrowattSensorEntityDescription(SensorEntityDescription, GrowattRequiredKey
     ),
 )
 
+TLX_SENSOR_TYPES: tuple[GrowattSensorEntityDescription, ...] = (
+    GrowattSensorEntityDescription(
+        key="tlx_energy_today",
+        name="Energy today",
+        api_key="eacToday",
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        device_class=DEVICE_CLASS_ENERGY,
+        precision=1,
+    ),
+    GrowattSensorEntityDescription(
+        key="tlx_energy_total",
```

Comment:
This one should have one of the new "TOTAL" measurement types on it. See the other PR for what to use.